### ISSUE TITLE
Failed to open sample videos.

### ISSUE THREAD

**Issue by hiccup0526** [RenderHeads/UnityPlugin-AVProVideo#1542]
- Unity version: 2022.2.15
 - AVPro Video version (number and edition (trial/core/ultra/enterprise)): Ultra
 - Operating system version: Win10
 - Device model: 
 - Video specs (resolution, frame-rate, codec, file size): StreamingAssets sample videos

Hello everyone. Thanks for nice this asset.
After importing ultra edition package, I tried to open sample videos that are in StreamingAssets.
When I play, I can see these logs.

[AVProVideo] Initialising AVPro Video v2.7.3 (native plugin v2.7.0f1-ultra) on NVIDIA GeForce RTX 3060/Direct3D 11.0 [level 11.1] (MT True) on WindowsEditor
[AVProVideo] Opening D:/Work/Unity/StreamingVideo/Assets/StreamingAssets\AVProVideoSamples/RenderHeads-1080p30-H264.mp4 (offset 0) with API MediaFoundation
[AVProVideo] Failed to open D:/Work/Unity/StreamingVideo/Assets/StreamingAssets\AVProVideoSamples/RenderHeads-1080p30-H264.mp4

I tried with untick "Hardware Decoding" with Media Foundation and Direct Show apis but it's same.
When I try with Win RT api, I got Runtime Error.

![image](https://github.com/RenderHeads/UnityPlugin-AVProVideo/assets/105547062/5991af27-e0f1-4597-b1d0-df9a62512755)

Please solve this problem.
Thanks.

Instead email the link to us mayeda.youichi0526@gmail.com

**Comment by Chris-RH** [comment]
I have just installed Unity 2022.2.15, started a new project with AVPro Video 2.7.3.
I opened the mediaplayer demo scene and tested the videos in the streaming assets folder using Media Foundation, Directshow and WinRT.  They all run as expected.

For reference, I'm using Win 10 22H2 and Geforce 1050 Ti.

Have you tried closing Unity and reimporting the asset?  
Are you testing in a new project using the demo scene?
Which version of Windows 10 are you using?
